.github/workflows/promote.yml

```diff
@@ -8,6 +8,15 @@ on:
         type: string
         required: false
         default: ''
+      worker_container_increment:
```

Comment:
only adding the option right now. TODO: Implement logic that actually does the version increment logic

Reply:
Thats fine